Patch to add a desktop entry.

Author: John Robert Beshir <namegduf@fudgeman.org>
Submitted Upstream: No. Patch is specific to the installation.
--- M1A1-old/AlephOne-MarathonInfinity.desktop	1969-12-31 21:00:00.000000000 -0300
+++ M1A1/AlephOne-MarathonInfinity.desktop	2008-06-10 17:06:11.000000000 -0300
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Encoding=UTF-8
+Version=1.0
+Name=Marathon Infinity - Aleph One
+Comment=A classic FPS on the Aleph One engine.
+GenericName=Classic FPS
+Exec=marathoninfinity
+Icon=@%Compile_goboShared%@/icons/hicolor/128x128/apps/AlephOne-MarathonInfinity.ico
+Terminal=false
+Type=Application
+Categories=Game;
+StartupNotify=true
